system_strings/sysmes_text.en

```diff
@@ -5459,159 +5515,255 @@ Death calls out to me. There's^nowhere left to run. At that^moment――
+// @flowchart_descr
+The situation is clear.
+ With my changed body, I'll break through the hordes of the Dead!
 }
 [sha:dbc487823e7270c0482527a53cffda8653cc20b6]{
 -- 鉄串の檻が逃げ道を塞ぐ。^目前には畏怖すべき吸血鬼。^なら、後は殺し合うだけ。
-鉄串の檻が逃げ道を塞ぐ。^目前には畏怖すべき吸血鬼。^なら、後は殺し合うだけ。
+// @flowchart_descr
+Behind me, a cage of iron pailings cuts off my escape.
+ Before me, there stands a fearsome vampire.
+ What is there to do but fight?
 }
 [sha:d397049850869ba90f56e4dc4a960c338bc21783]{
 -- 考えるより先に首を断った。^吸血鬼は死に、死者は土に還り、^贖罪だけが引き継がれた。
-考えるより先に首を断った。^吸血鬼は死に、死者は土に還り、^贖罪だけが引き継がれた。
+// @flowchart_descr
+I slice off her head faster than I can even think.
+ The vampire dies, and the Dead return to the earth.
+ Only atonement remains.
 }
 [sha:dc5c873d55f33089f0264e3ba62304849ae319c1]{
 -- 千載一遇の機会を見送った。^……この決着は、^彼女たちの物語だ。
-千載一遇の機会を見送った。^……この決着は、^彼女たちの物語だ。
+// @flowchart_descr
+I can't believe I let that once-in-a-lifetime opportunity pass me by...
+ Victory is already as good as hers.
 }
 [sha:575eca6331c79d09687eba643339289472ae0b0a]{
 -- 死人は際限なく群がってくる。^殺されるつもりはない。^必ず戻ると信じている。
-死人は際限なく群がってくる。^殺されるつもりはない。^必ず戻ると信じている。
+// @flowchart_descr
+The Dead keep flinging themselves at me with no end in sight.
+ But I don't plan on dying here――I have complete faith that she'll return.
 }
 [sha:613a32e9d2e7b11cfd4b51ac884a552e03e57d4a]{
 -- いまは、^夜に虹はかからなくとも。
-いまは、^夜に虹はかからなくとも。
+// @flowchart_descr
+Even if, as of now, there exist no rainbows in the night.
 }
 [sha:4ea15c8bcee2e5ac4c92cc71a477bf1266d38c99]{
 -- 代行者、復活。^断罪の杭は分け隔てなく^彼女たちの魂に。
-代行者、復活。^断罪の杭は分け隔てなく^彼女たちの魂に。
+// @flowchart_descr
+An Executor reborn.
+ The stake of condemnation resides equally within their hearts.
 }
 [sha:b5696b90166a79cb131cf846ffa27a0502ed1a10]{
 -- ……“もしも”の話は、^ここにはなく。
-……“もしも”の話は、^ここにはなく。
+// @flowchart_descr
+Here, now, there are no 'what-if's.
 }
 [sha:c62d546726abad7e02c846cf4cad0c97bfc83674]{
 -- 戦いは幕を下ろす。^月に照らされたその地には、^一部始終を見届けた姿があった。
-戦いは幕を下ろす。^月に照らされたその地には、^一部始終を見届けた姿があった。
+// @flowchart_descr
+The curtain falls on their battle.
+ From on high, a moonlit figure witnessed the entire affair.
 }
 [sha:869fbad78ab07f24a73961195dd4567dcc39e80c]{
 -- 戦いは終わり、^シエル先輩のアパートへ。^彼女への気持ちを再確認する。
-戦いは終わり、^シエル先輩のアパートへ。^彼女への気持ちを再確認する。
+// @flowchart_descr
+With the battle over, we head to Ciel's apartment.
+ I clearly state my feelings for her once more.
 }
 [sha:abb730b0ed1e127b6f69f6ecd1ed698925aed216]{
 -- 興味深い話題だったが、^深入りでできる空気でもない。^断腸の思いでスルー。
-興味深い話題だったが、^深入りでできる空気でもない。^断腸の思いでスルー。
+// Quoting 36610/36611.
+// This flowchart box is basically the entire file lol
+// @flowchart_descr
+Curious as I am, now isn't the time to be teasing Senpai.
+ As much as it pains me to do so, I have no choice but to ignore it for now.
 }
 [sha:16374b9263c04fe217bb4a7a67901be769435afd]{
 -- 今までの情報から逆算する。^つまり、先輩のそれは―――^その時から、もう既に―――
-今までの情報から逆算する。^つまり、先輩のそれは―――^その時から、もう既に―――
+// See 36614/5.
+// already [had that kind of figure]
+// @flowchart_descr
+Working backwards from what I know,
+ that means... even back then, Senpai already―――
```

Comment:
it doesn't sound as sane when cut off
either extend or change it to like "Senpai was already―――"